Readme.md

```diff
@@ -16,9 +15,11 @@ When a [DashboardView](https://docs.devexpress.com/eXpressAppFramework/DevExpres
 
 ## Implementation Details
 1. Add a new [ViewController](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.ViewController) to the **YourSolutionName.Module** project. For more information, refer to the following file: [DashboardFilterController.cs](./CS/EFCore/DependentDashboardEF/DependentDashboardEF.Module/Controllers/DashboardFilterController.cs).
-2. Access the target and source views.
-3. Handle the source view's list view selection changed event.
-4. Filter the target view in the event handler.
+2. In the `OnActivated` method, retrieve `DashboardViewItems` via the `FindItem` method. Also, subscribe to the `ControlCreated` event of a `DashboardViewItem` whose `ListView` is used to filter data (hereinafter referred to as `SourceView`).
+4. In the `ControlCreated` event handler retrieve the `SourceView` via the [DashboardViewItem.InnerView](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.Editors.DashboardViewItem.InnerView?p=netframework) property and subscribe to its [SelectionChanged](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.View.SelectionChanged?p=netframework) event.
+5. In the `SelectionChanged` event handler, retrieve the View to be filtered (hereinafter referred to as `TargetView`) in the same manner as in the previous step.
+6. To get an object to filter on, use the [ListView.CurrentObject](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.ListView.CurrentObject?p=netframework) property. This object must be loaded from `SourceView` Object Space to `TargetView` Object Space via the `GetObject` method.
```

Comment:
To get an object to filter on, use the <a href="https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.[View.SelectedObjects](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.View.SelectedObjects)">View.SelectedObjects</a> property. (the second sentence can be removed)

Reply:
```suggestion
6. To get an object to filter on, use the [ListView.CurrentObject](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.ListView.CurrentObject?p=netframework) property. 
```